gen/container_test.go

```diff
@@ -686,7 +686,7 @@ func TestCrazyTown(t *testing.T) {
 				},
 			},
 			wire.NewValueStruct(wire.Struct{Fields: []wire.Field{
-				{ID: 4, Value: wire.NewValueSet(
+				{ID: 4, Value: wire.NewValueList(
```

Comment:
@abhinav This change is somewhat less related to my change, it should have been `NewValueList` instead of `NewValueSet` since type [here](https://github.com/thriftrw/thriftrw-go/blob/dev/gen/internal/tests/containers/types.go#L25) is slice, I think it is something overlooked.

Reply:
`wire.Value` is the encoded representation of objects in ThriftRW.
Regardless of the language-level representation used, over-the-wire we
should say that this is a set as defined [here][1].

(It's a slice at the language-level because `set<string>` isn't
hashable: `map[map[string]struct{}]struct{}` isn't a valid map in Go.)

So this should be NewValueSet.

[1]: https://github.com/thriftrw/thriftrw-go/blob/dev/gen/internal/tests/thrift/containers.thrift#L32

Reply:
I see, that makes sense. I have now updated diff to undo the change.